src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -103,6 +110,12 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 			}));
 			this._validations.push(() => !this.required || this.editable || !!this._selectBox.value);
 		}
+
+		this._loadingBox = new SelectBox([this.getStatusText()], this.getStatusText(), this.contextViewService, this._loadingBoxContainer.nativeElement);
+		this._loadingBox.render(this._loadingBoxContainer.nativeElement);
+		this._register(this._loadingBox);
+		this._register(attachSelectBoxStyler(this._loadingBox, this.themeService));
+		this._loadingBoxContainer.nativeElement.className = ''; // Removing the dropdown arrow icon from the right
```

Comment:
Could you instead potentially add a new class that removes the dropdown arrow icon in CSS? NBD if not